js/login.js

```diff
@@ -0,0 +1,97 @@
+const emailInput = document.getElementById('email')
+const passwordInput = document.getElementById('password')
+const loginButton = document.querySelector('.login-form__button')
+const eyeicon = document.querySelector('.login-form__input--eye-icon')
+
+//Email Validation
+function validateEmail(email) {
+  const regex = /^[A-Za-z0-9_\.\-]+@[A-Za-z0-9\-]+/;
```

Comment:
정규표현식의 경우 따로 상수화 시켜주시지 않으면 해당 함수가 실행될 때마다 상수가 선언되어요!
밖으로 빼주시는 것을 추천드립니다!
```suggestion
  const REGEX = /^[A-Za-z0-9_\.\-]+@[A-Za-z0-9\-]+/;
```